cookbook/README.md

```diff
@@ -0,0 +1,96 @@
+# baragon-cookbook
+
+Installs Baragon (service or agent) on a node.  Baragon is the loadbalancer management service for [HubSpot's Singularity cluster management system](https://github.com/HubSpot/Singularity).
+
+## Supported Platforms
+
+Tested on an extensive range of platforms including Ubuntu 14.04 and nothing else.
+
+## Requirements
+
+### Zookeeper
+
+Baragon depends on Zookeeper to receive requests from Singularity and to relay requests from the service to the agent.  Using an existing Zookeeper cluster is totally fine if you happen to have one.
```

Comment:
I misremembered the nature of the connection between Singularity and Baragon at the time that I wrote this.  I'll fix it.